app/cypress/integration/reporter-access-all-pages.spec.js

```diff
@@ -33,7 +33,19 @@ describe('When logged in as a reporter', () => {
     cy.url().should('include', '/reporter/facilities');
     cy.get('tr');
     cy.contains('View Submitted').click();
-    cy.url().should('include', '/reporter/view-application');
+    cy.url().should(
+      'include',
+      `/reporter/application/${applicationId}/version/1/view`
+    );
     cy.get('.admin');
+
+    // test permanent redirection from the v1 view-application page
+    cy.visit(
+      `/reporter/view-application?applicationId=${applicationId}&version=1`
+    );
+    cy.url().should(
+      'include',
+      `/reporter/application/${applicationId}/version/1/view`
+    );
```

Comment:
We could add a `cy.get('#page-content');` in between to force cypress to wait and avoid timing issues, not sure how much of an issue this is in this case

Reply:
I wouldn't expect any issue here. The assertion has a timeout, and our server should redirect to the new URL quickly enough